### ISSUE TITLE
Veteran Touchpoints Review

### ISSUE THREAD

**Issue by denisecoveyduc** [department-of-veterans-affairs/va.gov-team#18512]
## Description

As a product team, we want to explore the various touch points that a veteran has to find information from the VA. 

## Background/Context/Resources

We know that veterans, their family and caregivers look for information in a variety of places. "Onsite" being on va.gov, "Offsite" meaning through google, bing and other browsers and "In person" meaning customer care touchpoints such as IRIS, Supports and Resources, Call Centers and Chatbot (coming soon).  We would like to discuss  1)what we know 2)What we don't know and 3) what we need to know about each of these various avenues.

The goal of this exercise is to better understand how these various avenues come together and what it may mean for our efforts to improve the search experience for veterans. 

---

## Tasks

- [x] Establish [Mural template](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1609522120977/130a85b6272c4134bd4add267f9a0242d7bf2187) to guide discussion
- [x] Conduct a brainstorming session to review the aspects outlined above for the Onsite search, Offsite search and In person experiences. [Planning to conduct session after user research sessions have  been completed] 
- [x] Summarize our findings and determine what product direction implications have been determined

## Definition of Done

- [x] Populate Venn Diagram in Mural

**Comment by denisecoveyduc** [comment]
Higher priority items have caused this to  move into sprint 44- meeting currently scheduled for March 25, 2021

**Comment by denisecoveyduc** [comment]
As a result of this exercise we were able to discuss the various search touch points a veteran has with the VA. We examined what we know about Onsite Search, Offsite Search and In Person (Call center etc) as well as what we don't know or how those experiences differ. 

Completed Mural can be seen here with summaries of each: https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1609522120977/130a85b6272c4134bd4add267f9a0242d7bf2187

Next Steps:

- We only considered this from the veteran's point of view and didn't explore other personas such as caregivers or family members. We could examine if their experiences differ from those of the veteran or how that informs what we know about each touch point. 
- We primarily thought of Offsite as the Google experience.  Adding additional thoughts from other offsite experiences (Bing and online forums (e.g. Reddit, Rallypoint.com, facebook etc.) could be an interesting addition. 
- @cindymerrill is pulling together some research questions for exploration. Sharing this information with her (and the 3 various search touch points) could be beneficial for her discussions. Any insights she is able to identify could help us understand potential opportunities for improvement.
